agent-releases/telegraf/1.13.2-darwin.json

```diff
@@ -0,0 +1,10 @@
+{
+  "type": "TELEGRAF",
+  "version": "1.13.2",
+  "labels": {
+    "agent_discovered_arch": "amd64",
+    "agent_discovered_os": "darwin"
+  },
+  "url": "https://homebrew.bintray.com/bottles/telegraf-1.13.2.high_sierra.bottle.tar.gz",
```

Comment:
You need to update!

Reply:
Oops, I have upgraded but missed that was part of the download URL. (I just did a text replacement of the version :) )

Reply:
Confirmed mojave URL works.